src/components/BatchTx/BatchTotalAmount.tsx

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
+import { Typography } from 'antd';
+import styled from 'styled-components';
+
+import { useBatchParser } from './useBatchParser';
+
+function BatchTotalAmount() {
+  const { parsedTxs } = useBatchParser();
+
+  const rowsWithKey = useMemo(() => {
+    let totalAmount = 0;
+    for (let row of parsedTxs) {
+      let amount = Number(row['amount']?.formattedValue.split(',').join('').split('$').join(''));
```

Comment:
- `const`
- `row.amount`
- you're repeating the `split ... join` logic here again so extract it into a common function
- OR use `parsedValue` instead, which is even more correct - 99.9% of the time you should always do data manipulation on parsed values and only use formatted ones to present the data to the user
- in case of `parsedValue` you will also need the decimals from the selected asset to properly format the value